conan/cli/command.py

```diff
@@ -63,11 +66,13 @@ def _help_formatters(self):
         return [formatter for formatter in self._formatters if formatter != "text"]
 
     def _init_formatters(self, parser):
-        formatters = self._help_formatters
-        if formatters:
-            help_message = "Select the output format: {}".format(", ".join(formatters))
+        if self._help_formatters:
+            help_message = "Select the output format: {}".format(", ".join(self._help_formatters))
             parser.add_argument('-f', '--format', action=OnceArgument, help=help_message)
 
+        parser.add_argument("--out-file", action=OnceArgument,
```

Comment:
finally keeping this unconditionally as a direct replacement to redirect output with `>`